scripts/main.js

```diff
@@ -71,4 +71,135 @@ const departments = {
-console.log(departments);
\ No newline at end of file
+// 1.
+console.log("De afdeling Sales heeft " + departments.sales.numberOfEmployees + " medewerkers.");
+console.log("Marketing is een leuke afdeling om te werken. " + departments.marketing.description);
+console.log("De afdeling Customer Service heeft " + departments['customer-service'].numberOfEmployees + " medewerkers.");
+console.log("Sales is een uitdagende afdeling om te werken als " + departments.sales.jobs[1].title + ". " + departments.sales.jobs[1].description);
+
+// 2, 3 en 4 samengevoegd
+let userInput = prompt("Over welke afdeling wil je meer informatie? Kies uit: [1: marketing / 2: sales / 3: customer-service]");
+
+if (userInput === "marketing" || userInput === "1") {
+    console.log("Je koos " + userInput + ". " + departments.marketing.description);
+    userInput = prompt("Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in.  " +
+        "0: " + departments.marketing.jobs[0].title + "  " +
+        "1: " + departments.marketing.jobs[1].title + "  " +
+        "2: " + departments.marketing.jobs[2].title + "  " +
+        "3: " + departments.marketing.jobs[3].title);
+    switch (userInput) {
+        case "0":
+            console.log("Je koos " + departments.marketing.jobs[0].title + ". " + departments.marketing.jobs[0].description);
+
+            document.getElementById("role-title").textContent = departments.marketing.jobs[0].title;
+            document.getElementById("role-description").textContent = departments.marketing.jobs[0].description;
+            break;
+        case "1":
+            console.log("Je koos " + departments.marketing.jobs[1].title + ". " + departments.marketing.jobs[1].description);
+
+            document.getElementById("role-title").textContent = departments.marketing.jobs[1].title;
+            document.getElementById("role-description").textContent = departments.marketing.jobs[1].description;
+            break;
+        case "2":
+            console.log("Je koos " + departments.marketing.jobs[2].title + ". " + departments.marketing.jobs[2].description);
+
+            document.getElementById("role-title").textContent = departments.marketing.jobs[2].title;
+            document.getElementById("role-description").textContent = departments.marketing.jobs[2].description;
+            break;
+        case "3":
+            console.log("Je koos " + departments.marketing.jobs[3].title + ". " + departments.marketing.jobs[3].description)
+
+            document.getElementById("role-title").textContent = departments.marketing.jobs[3].title;
+            document.getElementById("role-description").textContent = departments.marketing.jobs[3].description;
+            break;
+        default:
+            console.log("Ongeldige invoer. Ververs de pagina en probeer het opnieuw.")
+            document.getElementById("error-message").textContent = "Ongeldige invoer. Ververs de pagina en probeer het opnieuw.";
+            break;
+    }
```

Comment:
Ik zie deze switch telkens herhaald worden per afdeling. Wist je dat je met `departments[userInput]` hetzelfde kan bereiken als `departments.marketing` (mits `userInput` de waarde "marketing" heeft).

Als je dit weet, dan hoef je deze switch maar één keer te hebben 😉 Net zoals in de uitwerkingen: https://github.com/hogeschoolnovi/frontend-javascript-job-prompter-uitwerkingen/blob/uitwerkingen/scripts/main.js

Reply:
En als je dan zowel `marketing` als `1` wilt toestaan, dan moet je ervoor zorgen dat `"1"` uiteindelijk vertaald wordt naar `marketing`:
```
if (userInput === "1") userInput = "marketing";
```

Reply:
Goede tip. Thanks!